src/models/Item.ts

```diff
@@ -81,6 +82,10 @@ export default class Item {
       .includes("all")
   }
 
+  get holdRequestSource(): string {
```

Comment:
this sounds like it means where the hold request is coming from (which is something that we think about on LSP, though not necessarily in this app). to avoid confusion, maybe `formattedSourceForHoldRequest`

Reply:
good call! done